### PR TITLE
Microsoft.AzureDataCLI version 20.3.5

### PR DESCRIPTION
**Pull request by tamckenna**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17005)

### CODE REVIEW

File: manifests/m/Microsoft/AzureDataCLI/20.3.5/Microsoft.AzureDataCLI.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: Microsoft.AzureDataCLI
+PackageVersion: 20.3.5
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral
+  InstallerType: msi
+  InstallerUrl: https://aka.ms/azdata-msi
```
Comment (OfficialEsco) [inline manifests/m/Microsoft/AzureDataCLI/20.3.5/Microsoft.AzureDataCLI.installer.yaml:9]:
```suggestion
  InstallerUrl: https://download.microsoft.com/download/f/f/f/fffaa914-d4f7-4885-89c7-696bbfe7670a/azdata-cli-20.3.5.msi
```
Static URL

File: manifests/m/Microsoft/AzureDataCLI/20.3.5/Microsoft.AzureDataCLI.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: Microsoft.AzureDataCLI
+PackageVersion: 20.3.5
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral
```
Comment (OfficialEsco) [inline manifests/m/Microsoft/AzureDataCLI/20.3.5/Microsoft.AzureDataCLI.installer.yaml:7]:
```suggestion
  Architecture: x32
```

File: manifests/m/Microsoft/AzureDataCLI/20.3.5/Microsoft.AzureDataCLI.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Microsoft.AzureDataCLI
+PackageVersion: 20.3.5
+PackageLocale: en-US
+Publisher: Microsoft
```
Comment (OfficialEsco) [inline manifests/m/Microsoft/AzureDataCLI/20.3.5/Microsoft.AzureDataCLI.locale.en-US.yaml:6]:
```suggestion
Publisher: Microsoft Corporation
```
Needs to match what control panel sees.

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/m/Microsoft/AzureDataCLI/20.3.5/Microsoft.AzureDataCLI.locale.en-US.yaml (hunk 3 above)
Comment (tamckenna) [reply manifests/m/Microsoft/AzureDataCLI/20.3.5/Microsoft.AzureDataCLI.locale.en-US.yaml]: Thanks for the suggestions and reasons behind them! @OfficialEsco

File: manifests/m/Microsoft/AzureDataCLI/20.3.5/Microsoft.AzureDataCLI.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: Microsoft.AzureDataCLI
+PackageVersion: 20.3.5
+Installers:
+- InstallerLocale: en-US
+  Architecture: x32
```
Comment (OfficialEsco) [inline manifests/m/Microsoft/AzureDataCLI/20.3.5/Microsoft.AzureDataCLI.installer.yaml:7]:
```suggestion
  Architecture: x86
```
Ooops my bad 🤦‍♂️

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Pull request closed without merging**